lib/msf/core/opt_base.rb

```diff
@@ -57,6 +57,7 @@ def initialize(in_name, attrs = [],
       end
 
       unless max_length.nil?
+        self.desc ||= ''
```

Comment:
Any context on this change? 👀

Reply:
After some testing, this is no longer necessary. I've removed it and will push the changes in a second 💯  Thanks for pointing this out.